server/sonar-server/src/test/resources/org/sonar/server/db/migrations/v52/FeedProjectLinksComponentUuidTest/schema.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE "PROJECT_LINKS" (
```

Comment:
In fact I prefer let them all, in the case we need another column, we don't need to update tests.